inc/section.class.php

```diff
@@ -81,11 +81,6 @@ public static function getTypeName($nb = 0) {
    public function prepareInputForAdd($input) {
       global $DB;
 
-      // Decode (if already encoded) and encode strings to avoid problems with quotes
-      foreach ($input as $key => $value) {
-         $input[$key] = plugin_formcreator_encode($value);
-      }
-
```

Comment:
only the columns ```name``` may contain risky chars (uuid is generated programatically)

![image](https://user-images.githubusercontent.com/14139801/37904474-299c6694-30fc-11e8-98f7-e6c076d28900.png)